### PR TITLE
spelling mistake in firefighter mech

### PR DESCRIPTION
**Pull request by hal9000PR**
## What Does This PR Do
fixes a spelling mistake in the firefighter mechs description 

## Why It's Good For The Game
spelling good

## Changelog
:cl:
fix: spelling in the firefighters description
/:cl:

### CODE REVIEW

File: code/game/mecha/working/ripley.dm [hunk 1]
```diff
@@ -62,7 +62,7 @@
 			add_overlay(occupant ? "ripley-g-full" : "ripley-g-full-open")
 
 /obj/mecha/working/ripley/firefighter
-	desc = "Standart APLU chassis was refitted with additional thermal protection and cistern."
+	desc = "Standard APLU chassis was refitted with additional thermal protection and cistern."
```
Comment (S34NW) [inline code/game/mecha/working/ripley.dm:65]:
```suggestion
	desc = "Standard APLU chassis that was refitted with additional thermal protection and cistern."
```

This description makes no sense by itself

**Review by S34NW — CHANGES_REQUESTED** [review]

File: code/game/mecha/working/ripley.dm (hunk 1 above)
Comment (SabreML) [reply code/game/mecha/working/ripley.dm:65]: May as well do the whole thing
```suggestion
	desc = "Standard APLU chassis that was refitted with additional thermal protection and a cistern."
```

Comment (hal9000PR) [reply code/game/mecha/working/ripley.dm:65]: hmm yes this spelling mistake is made out of spelling mistake

Comment (S34NW) [reply code/game/mecha/working/ripley.dm:65]: Do you think there should be an "A" at the start too?

**Review by SteelSlayer — APPROVED** [review]

**Review by AffectedArc07 — APPROVED** [review]

**Pull request merged**